src/test/java/org/broadinstitute/hellbender/tools/exome/DecomposeSingularValuesIntegrationTest.java

```diff
@@ -86,15 +86,30 @@ private void assertSVDValues(final File outputFileV, final File outputFileS, fin
             assertOutputFileValues(outputFileU, svd.getU());
             assertOutputFileValues(outputFileS, sDiag);
             assertOutputFileValues(outputFileV, svd.getV());
-            SVDUnitTest.assertUnitaryMatrix(svd.getV());
-            SVDUnitTest.assertUnitaryMatrix(svd.getU());
+            assertUnitaryMatrix(svd.getV());
+            assertUnitaryMatrix(svd.getU());
             Assert.assertTrue(MatrixUtils.isSymmetric(sDiag, 1e-32));
 
         } catch (final IOException ioe) {
             Assert.fail("Could not open test file: " + CONTROL_PCOV_FULL_FILE, ioe);
         }
     }
 
+    /**
+     * Assrt that the given matrix is unitary.
+     * @param m
+     */
+    public static void assertUnitaryMatrix(final RealMatrix m){
+        final RealMatrix mInv = new QRDecomposition(m).getSolver().getInverse();
```

Comment:
Same answer as there -- Apache Commons has a weird SVD, but nothing we can do about it.